src/index.ts

```diff
@@ -123,7 +130,12 @@ export async function isAvailable() {
     // Attempt to read instance metadata. As configured, this will
     // retry 3 times if there is a valid response, and fail fast
```

Comment:
Is the `retry 3 times` part still valid?

Reply:
good point, I'll change that comment.

Reply:
I removed the comment, as it's stale.